tasks/xpack/security/elasticsearch-security-native.yml

```diff
@@ -49,9 +67,20 @@
   set_fact: change_api_password=true
   when: manage_native_users and es_api_basic_auth_username in native_users and native_users[es_api_basic_auth_username].password is defined
 
+- name: List Native Users
```

Comment:
Could you give this a different task name? It took me a little while to figure out the difference between this task and the ones above since they are very similar.